### PR TITLE
[3] GET /api/user/id_user

### LINKED ISSUE #3: GET /api/user/id_user  Id usuario obligatorio (opcional):
• call:: Get /api/user
• response: { ‘error’ : ‘El id de usuario es obligatorio’}

### PR DESCRIPTION
**Pull request by jalex3421**
fixed

### CODE REVIEW

**Comment by crackergirl** [comment]
it's all OK.

File: app/Infrastructure/Controllers/GetUserController.php [hunk 1]
```diff
@@ -13,9 +13,15 @@ class GetUserController extends BaseController
 
     public function __invoke(string $userId): JsonResponse
     {
+        if(strlen($userId)<1){
```
Comment (crackergirl) [inline app/Infrastructure/Controllers/GetUserController.php:16]: Please, if you have created this piece of code in another class, delete it as soon as possible.

**Review by crackergirl — COMMENTED** [review]
Please, fix it.

File: app/Infrastructure/Controllers/GetUserController.php (hunk 1 above)
Comment (jalex3421) [reply app/Infrastructure/Controllers/GetUserController.php:16]: I have fixed it right now madam

**Pull request merged**